client/actions/users.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import { USER_CONFIGS } from './types';
+
+export const getAllUsersDispatch = payload => ({
+    type: USER_CONFIGS,
+    payload
+});
+
+export function getAllUsers() {
```

Comment:
I was thinking this should be the api you use to get users.. get user config is perhaps misnamed on the backend. If you want all the users in the system for some reason (as an admin, to override their settings, or to know who can test what) you would just use this endpoint. It will come with their configs by default, but you need that data almost every time  you need the list of users anyway.